### PR TITLE
Gain in orthogonal initialization

### PR DESCRIPTION
**Pull request by janchorowski**

### CODE REVIEW

**Comment by dmitriy-serdyuk** [comment]
Is it a common name, `gain`?

**Comment by janchorowski** [comment]
I was referring to: http://arxiv.org/pdf/1312.6120v3.pdf though he introduces another gain term. Maybe scale would be better. Anyway, it is a useful parameter.

**Comment by dmitriy-serdyuk** [comment]
Yeah, it is useful, I would call it `scale`.

**Comment by janchorowski** [comment]
renamed

File: blocks/initialization.py [hunk 1]
```diff
@@ -159,6 +159,9 @@ class Orthogonal(NdarrayInitialization):
     Only works for 2D arrays.
 
     """
+    def __init__(self, scale=1):
```
Comment (rizar) [inline blocks/initialization.py]: Please add the new parameters to the docstring. Literature reference would be appreciated as well.

**Comment by janchorowski** [comment]
Added doc

**Comment by rizar** [comment]
I am sorry for not spotting everything from the first look, but it also needs a unit-test. Could you please make one?

**Comment by janchorowski** [comment]
Sorry for forgetting about them in the first place. Here they are

**Comment by janchorowski** [comment]
The failing test is unrelated to this change - what do I do about it?

**Comment by rizar** [comment]
Unfortunately we seems to have a bit of stochasticity in our tests :)

**Pull request merged**